speculos/api/events.py

```diff
@@ -27,6 +28,12 @@ def remove_client(self, client):
 
     def broadcast(self, event):
         self.logger.debug(f"events: broadcasting {event} to ({len(self.clients)}) client(s)")
+        if self.screen_content:
+            y_trigger = self.screen_content[-1]["y"] + 10
```

Comment:
Can you add a comment to explain the purpose of this `10` and how the value was chosen?

Reply:
Ok